### PR TITLE
[MIG] stock_picking_operation_quick_change to 14.0

### PR DESCRIPTION
**Pull request by sergiocorato**

### CODE REVIEW

**Comment by rousseldenis** [comment]
@sergiocorato Could you fix this as you should have two commits (one for pre-commit, one for migration)? Thanks

**Comment by sergiocorato** [comment]
> @sergiocorato Could you fix this as you should have two commits (one for pre-commit, one for migration)? Thanks

~I haven't put two commits because I haven't done any migration change to the code.~ I'll create a commit, thanks.

**Comment by sergiocorato** [comment]
@rousseldenis Could you re-check the PR?

File: stock_picking_operation_quick_change/__manifest__.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 {
     "name": "Stock Picking Operation Quick Change",
     "summary": "Change location of all picking operations",
-    "version": "13.0.1.1.0",
+    "version": "14.0.1.0.0",
```
Comment (rousseldenis) [inline stock_picking_operation_quick_change/__manifest__.py:7]: That one in migration commit :smiley:

Comment (sergiocorato) [reply stock_picking_operation_quick_change/__manifest__.py:7]: It seems ok now :)

**Pull request closed without merging**